api/data_refinery_api/views.py

```diff
@@ -327,6 +332,16 @@ class ExperimentDocumentView(DocumentViewSet):
             'enabled': True,
             'global': False,
         },
+        'num_downloadable_samples__gt': {
+            'field': 'num_downloadable_samples',
+            'facet': RangeFacet,
+            'options': {
+                'ranges': [
+                    ("0", (0, None)),
```

Comment:
I'm not sure how ranges work with elasticsearch, but does this range exclude 0? At a first glance it looks like it might include 0.

Reply:
Good catch, just confirmed in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.1/search-aggregations-bucket-range-aggregation.html)

> this aggregation includes the from value and excludes the to value for each range.